src/auth-service/utils/email.msgs.js

```diff
@@ -1,34 +1,35 @@
 module.exports = {
-    confirm: "Email sent, please check your inbox to confirm",
-    confirmed: "Your email is confirmed!",
-    resend: "Confirmation email resent, maybe check your spam?",
-    couldNotFind: "Could not find you!",
-    alreadyConfirmed: "Your email was already confirmed",
-    recovery_email: (token) => {
-        return (
-            "You are receiving this because you (or someone else) have requested the reset of the password for your account.\n\n" +
-            "Please click on the following link, or paste this into your browser to complete the process within one hour of receiving it:\n\n" +
-            `http://localhost:5000/reset/${token}\n\n` +
```

Comment:
This line is what was changed.....because people were receiving localhost for password in production. @sserurich